src/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.cpp

```diff
@@ -205,13 +205,14 @@ namespace NovelRT::ResourceManagement::Desktop
             returnImage.emplace_back(rawImage[i]);
         }
 
+        delete[] rawImage;
+        delete[] data.rowPointers;
```

Comment:
Why are we deleting the row pointers?

Reply:
because we don't need them at this point.